pkg/minikube/node/config.go

```diff
@@ -79,3 +77,34 @@ func configureMounts(wg *sync.WaitGroup, mount bool, mountString string) {
 		exit.Error(reason.HostMountPid, "Error writing mount pid", err)
 	}
 }
+
+func generateMountArgs(profile string, cc config.ClusterConfig) []string {
+	mountDebugVal := 0
+	if klog.V(8).Enabled() {
+		mountDebugVal = 1
+	}
+
+	args := []string{"mount", cc.MountString}
+	flags := []struct {
+		name  string
+		value string
+	}{
+		{"profile", profile},
+		{"v", fmt.Sprintf("%d", mountDebugVal)},
+		{constants.Mount9PVersionFlag, cc.Mount9PVersion},
+		{constants.MountGIDFlag, cc.MountGID},
+		{constants.MountIPFlag, cc.MountIP},
+		{constants.MountMSizeFlag, fmt.Sprintf("%d", cc.MountMSize)},
+		{constants.MountModeFlag, fmt.Sprintf("%d", cc.MountMode)},
+		{constants.MountPortFlag, fmt.Sprintf("%d", cc.MountPort)},
+		{constants.MountTypeFlag, cc.MountType},
+		{constants.MountUIDFlag, cc.MountUID},
+	}
+	for _, flag := range flags {
+		args = append(args, fmt.Sprintf("--%s", flag.name), flag.value)
+	}
+	for _, option := range cc.MountOptions {
+		args = append(args, fmt.Sprintf("--%s", constants.MountOptionsFlag), option)
+	}
+	return args
```

Comment:
Are there any cases where flags could possess inconsistent values?

Should we have logic, to say, for example, that mountSize value is out of range (maybe we want to do this in a separate PR).

Reply:
Yeah, I can add that is a follow up PR, I created an issue

https://github.com/kubernetes/minikube/issues/13142

Reply:
Makes sense!